.github/workflows/release.yml

```diff
@@ -18,9 +18,9 @@ defaults:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I like this. This way we don't have to worry about suddenly the build system breaks anymore :+1:

Reply:
> ```
> release:
>     if: startsWith(github.ref, 'refs/tags/v')
>     runs-on: ubuntu-latest
>     needs: build
> ```
> 
> Do we need to change machine in the release action?

Yes we do. Thanks for catching that.